src/hooks/useWebSocket.ts

```
@@ -60,13 +64,21 @@ export const useWebSocket = (options: UseWebSocketOptions = {}) => {
+
+        onSumsubKycStatusUpdate,
         onTosUpdate,
         onPendingPerk,
+
         onConnect,
+
         onDisconnect,
+
         onError,
+        ,
     ])
```

Biome (2.3.14)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)